### PR TITLE
Fix typos/grammar

### PR DESCRIPTION
**Pull request by christophrumpel**
This PR fixes a few typos in the Eventy announcement article. We can probably argue about some of the changes so feel free to only use what fits you.

### CODE REVIEW

File: source/_posts/announcing-eventy.md [hunk 1]
```diff
@@ -4,33 +4,33 @@ section: content
 title: Announcing Eventy
 date: 2019-10-19
 ---
-Last Tuesday I finally announced my next passion project, [Eventy](https://eventy.io). I must say that the timing was a little off. I've been planning on doing the announcement for several weeks now but every time something kept creeping in between. So when I saw the announcement by Meetup.com's new pricing plans and all of the outrage by that, I knew that there wouldn't be a better moment then that.
+Last Tuesday, I finally announced my next passion project, [Eventy](https://eventy.io). I must say that the timing was a little off. I've been planning on doing the announcement for several weeks now, but every time something kept creeping in between. So when I saw the announcement by Meetup.com's new pricing plans and all of the outrages by that, I knew that there wouldn't be a better moment than that.
```
Comment (driesvints) [inline source/_posts/announcing-eventy.md]: I think "outrage" is still correct here.

File: source/_posts/announcing-eventy.md [hunk 2]
```diff
@@ -4,33 +4,33 @@ section: content
...
 
-Eventy is an app that I've been thinking of building for quite a while now. But over the past years the idea has been becoming more and more concrete. The thing that I sorely miss, being a user group leader, conference organiser, speaker and attendee is a unified platform. A place where I can maintain and keep track of all these different parts of an event experience. All while having a really nice end user experience.
+Eventy is an app that I've been thinking of building for quite a while now. But over the past years, the idea has been becoming more and more concrete. The thing that I sorely miss, being a user group leader, conference organizer, speaker, and attendee, is a unified platform. A place where I can maintain and keep track of all these different parts of an event experience. All while having a really nice end-user experience.
```
Comment (driesvints) [inline source/_posts/announcing-eventy.md]: Organiser is the British spelling while Organizer is the US one so it's actually still correct. But I don't mind editing it here

File: source/_posts/announcing-eventy.md [hunk 3]
```diff
@@ -4,33 +4,33 @@ section: content
...
 
-The landscape for event software is too scattered at the moment. You have so many different kind of apps which might excel in what they do but they don't work together. So you'll have to maintain multiple accounts and switch apps all the time. 
+The landscape for event software is too scattered at the moment. You have so many different kinds of apps that might excel in what they do, but they don't work together. So you'll have to maintain multiple accounts and switch apps all the time.
```
Comment (driesvints) [inline source/_posts/announcing-eventy.md]: I think "kind" is also correct here.

**Comment by driesvints** [comment]
@christophrumpel Thank you!! 🙏 

I left a couple of notes.

**Comment by driesvints** [comment]
Went ahead and reverted the two small changes. Thanks for helping!

**Comment by christophrumpel** [comment]
Ah sry forgot to reply to your comments. Glad it helped 👍

**Pull request merged**